content/en/docs/concepts/services-networking/service.md

```diff
@@ -1100,21 +1103,15 @@ about the API object at: [Service API object](/docs/reference/generated/kubernet
 
 ### TCP
 
-{{< feature-state for_k8s_version="v1.0" state="stable" >}}
-
 You can use TCP for any kind of Service, and it's the default network protocol.
 
 ### UDP
 
-{{< feature-state for_k8s_version="v1.0" state="stable" >}}
-
 You can use UDP for most Services. For type=LoadBalancer Services, UDP support
 depends on the cloud provider offering this facility.
 
 ### HTTP
 
-{{< feature-state for_k8s_version="v1.1" state="stable" >}}
```

Comment:
If dropping these, there is also a feature-state shortcode for PROXY protocol. Maybe drop that as well?

Reply:
Good spot. Fixed.